app/src/main/res/values/strings.xml

```diff
@@ -823,5 +823,8 @@ Please report any problems you encounter via Github.</string>
     <string name="browse">Browse</string>
     <string name="no_sub_folder_is_selected">No sub folder is selected</string>
     <string name="share_device_id_chooser">Share device ID with</string>
+    <string name="require_notification_permission_desc">Syncthing requires your permission to show you notifications.</string>
```

Comment:
This should describe why syncthing needs/wants to show the user notifications. Maybe something like this:

```suggestion
    <string name="require_notification_permission_desc">Syncthing requires your permission to show you notifications. Those are used to let you know about a new device connecting, or a new folder sharing request.</string>
```

Reply:
sure think looks more descriptive. let me add this.